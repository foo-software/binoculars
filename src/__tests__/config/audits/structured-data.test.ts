import auditStructuredData from '../../../config/audits/structured-data';

describe('structured-data audit', () => {
  it('should match snapshot', () => {
    expect(auditStructuredData('en')).toMatchSnapshot();
  });
});
