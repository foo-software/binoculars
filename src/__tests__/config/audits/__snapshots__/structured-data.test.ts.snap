// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`structured-data audit should match snapshot 1`] = `
Object {
  "description": "Run the [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/) and the [Structured Data Linter](http://linter.structured-data.org/) to validate structured data. [Learn more](https://web.dev/structured-data/).",
  "title": "Structured data is valid",
}
`;
